### DISCUSSION TITLE
Add full support for Lifelynk

### DISCUSSION THREAD

**Discussion by jon0881** [slipx06/sunsynk-power-flow-card#484] [Lifelynk]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Almost all values are matched and pulled from the Sunsynk connect portal but there seems to be a missing value when it comes to pulling the grid and load values. I can see the values in the sunsynk connect app but the matched values in the card in HA are 0 value

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/54940202/71146e76-f922-4596-93e7-75e1f86818c5)

### Expected behaviour

grid and load values to be populated

### Mode

Both

### Context / Reason

maybe have lifelynk as an inverter option if the data needs to be matched to a different entity?

**Comment by slipx06** [comment]
Hi. The card will display the provided sensor. Make sure you have the correct sensors defined for:

```
grid_ct_power_172:
essential_power:
```

**Comment by slipx06** [comment]
Also if you have a set of Lifelynk inverter codes and an image you would like displayed please share

**Reply by RenierBoshoff** [reply]
Following...

**Comment by jon0881** [comment]
Im happy to share those but im not sure where to find them. I did log a ticcket with Sunsynk as i think even in thier portal some data is not matching to the right place. For example on the input you can see power and PV1 power but not the volts or the amps
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/54940202/735556fd-d22e-45ca-82f1-1d710004b5dd)
